autospec/tarball.py

```diff
@@ -471,7 +471,8 @@ def name_and_version(name_arg, version_arg, filemanager):
         buildreq.add_buildreq("buildreq-gnome")
 
     if "kde.org" in url or "https://github.com/KDE" in url:
-        buildreq.add_buildreq("buildreq-kde")
+        if "kxmlgui" not in url:
+            buildreq.add_buildreq("buildreq-kde")
```

Comment:
I assume there is some ordering issue preventing kxmlgui from buildreq_banning buildreq-kde?

Reply:
Probably, I don't know why buildreq_ban does not apply.

Reply:
It is an ordering issue... a PR will be opened shortly.

Reply:
With @phmccarty change merged I'm closing this as buildreq_ban will apply to urls now too.